src/mahoji/lib/abstracted_commands/infernoCommand.ts

```diff
@@ -163,13 +161,19 @@ async function infernoRun({
 			.map(([name, lvl]) => `${lvl} ${name}`)
 			.join(', ')}.`;
 	}
+
 	/**
 	 *
 	 * Item Requirements
 	 *
 	 */
-	if (!user.owns(itemRequirements)) {
-		return `To do the Inferno, you need these items: ${itemRequirements}.`;
+	const itemRequirements = getSimilarItems(itemID('Rune pouch'));
+	if (itemRequirements.every(item => !user.owns(item))) {
+		let itemRequirementNames: string[] = [];
+		for (const item of itemRequirements) {
+			itemRequirementNames.push(itemNameFromID(item) as string);
+		}
+		return `To do the Inferno, you need one of these items: ${itemRequirementNames.join(', ')}.`;
```

Comment:
this `itemRequirementNames` is a bit odd. Try just `itemRequirements.map(itemNameFromID).join(", ")`